site/gatsby-site/migrations/2022.06.23T20.09.47.initialize-empty-nlp-similar-incidents.js

```diff
@@ -0,0 +1,39 @@
+const config = require('../config');
+
+/** @type {import('umzug').MigrationFn<any>} */
+exports.up = async ({ context: { client } }) => {
+  const submissionsCollection = client
+    .db(config.realm.production_db.db_name)
+    .collection('submissions');
+
+  await submissionsCollection.updateMany(
```

Comment:
Sometimes `updateMany` gives me problems. Have you experienced any issues? Otherwise, this looks good.

Reply:
I haven't had any problems with it here, but if anyone does, I'll switch to using batched `updateOne`s as used in some of the other migrations.